eng/liveBuilds.targets

```diff
@@ -122,8 +127,14 @@
         <TargetPath>runtime/$(CoreCLRCrossTargetComponentDirName)_$(TargetArchitecture)/native</TargetPath>
       </CoreCLRCrossTargetFiles>
     </ItemGroup>
+    <ItemGroup Condition="'$(RuntimeFlavor)' == 'Mono'">
+      <CoreCLRFiles Include="$(MonoArtifactsPath)\*.*" />
+      <CoreCLRFiles>
+        <IsNative>true</IsNative>
+      </CoreCLRFiles>
+    </ItemGroup>
 
-    <Error Condition="'@(CoreCLRFiles)' == ''" Text="The CoreCLR subset category must be built before building this project." />
+    <Error Condition="'@(CoreCLRFiles)' == ''" Text="The $(RuntimeFlavor) subset category must be built before building this project." />
   </Target>
 
   <Target Name="ResolveMonoFilesFromLocalBuild">
```

Comment:
Do we still need this target?

Reply:
If we also change `ResolveMonoFilesFromLocalBuild` to `ResolveRuntimeFilesFromLocalBuild` or even a more concise way to combine the two in `eng/referenceFromRuntime/targets`?

Reply:
I wasn't sure how big of a bite to take.  We would take away `ResolveMonoFilesFromLocalBuild` and probably want to change `<CoreCLRFiles>` too